backend/qemu.pm

```diff
@@ -761,6 +761,11 @@ sub start_qemu {
     sp('serial',  'file:serial0');
     sp('soundhw', 'ac97');
     {
+        # Remove floppy drive device on archictures
+        unless ($vars->{ARCH} eq 'aarch64' || $vars->{ARCH} eq 'arm' || $vars->{QEMU_NO_FDC_SET}) {
+            sp('global', 'isa-fdc.driveA=');
```

Comment:
Just for future reference https://github.com/qemu/qemu/blob/master/docs/qdev-device-use.txt#L107